mlflow/server/js/src/components/App.js

```diff
@@ -42,7 +42,7 @@ class App extends Component {
             </div>
           </header>
           <AppErrorBoundary>
-            { /* Since the experiment list side bar goes outside of the 80% width put outside of div */ }
+            { /* Since the experiment sidebar goes outside of the 80% width, put outside of div */ }
```

Comment:
Did the linter check our grammar too :)

Reply:
No but it checked for >100 character lines!